azure-quantum/azure/quantum/optimization/problem.py

```diff
@@ -124,12 +127,46 @@ def add_term(self, c: Union[int, float], indices: List[int]):
         self.terms.append(Term(indices=indices, c=c))
         self.uploaded_blob_uri = None
 
-    def add_terms(self, terms: List[Term]):
-        """Adds a list of terms to the `Problem` representation
+    def add_terms(self, terms: List[Term],
+                  type: str = None, c: Union[int, float] = 1):
```

Comment:
consider creating an enum

Reply:
I notice you already have GroupType - can we use that as an optional input? i.e. Optional[GroupType]

Reply:
I allowed for type to be either GroupType or str to leverage the specificity one with the flexibility of the other.